salt/etc-hosts/init.sls

```diff
@@ -13,3 +13,13 @@ dummy_step:
   cmd.run:
     - name: "echo saltstack bug 14553"
 {% endif %}
+
+{% if "admin" in salt['grains.get']('roles', []) %}
+update-velum-hosts:
+  cmd.run:
+    - name: |-
+        velum_id=$( docker ps | grep velum-dashboard | awk '{print $1}')
+        docker cp /etc/hosts $velum_id:/etc/hosts
```

Comment:
should we include a check if the container is really up and running? i mean some protection if `velum_id` is empty

Reply:
Done, thank you Max